controllers/module_controller_test.go

```diff
@@ -23,7 +23,7 @@ var _ = Describe("Workspace controller", Ordered, func() {
 
 	BeforeAll(func() {
 		// Set default Eventually timers
-		SetDefaultEventuallyTimeout(120 * time.Second)
+		SetDefaultEventuallyTimeout(syncPeriod * 4)
```

Comment:
This is also not relevant to the issue, just an improvement I made while working on tests.